charts/camunda-platform/test/unit/optimize/golden/deployment.golden.yaml

```diff
@@ -39,7 +39,7 @@ spec:
         []
       initContainers:
         - name: migration
-          image: camunda/optimize:3.11.1
+          image: camunda/optimize:8.3.1
```

Comment:
The global section already has a tag. Since this version conforms to the tag on the global section, I think it would make more sense to leave the tag empty for optimize so it grabs the tag from the global section.

Reply:
It's a good thought, but `global.image.tag` will actually need to be deprecated in the near future as the application components will no longer be in sync in 8.3.  @hisImminence can confirm

Reply:
It's a good thought, but `global.image.tag` will actually need to be deprecated in the near future as the application components will no longer be in sync in 8.3.  @hisImminence can confirm

Reply:
If that's the case then it's good to merge. I'll approve. We can merge when @hisImminence has confirmed.

Reply:
> It's a good thought, but global.image.tag will actually need to be deprecated in the near future

I assume @jessesimpson36 refers to this ticket: https://github.com/camunda/product-hub/issues/1699
tltr: Its correct that our plan is to decouple the HC version from the application versions. So far I have no date yet but many evidence we should work on this rather sooner than later 🙈 

What ever feels easier to you/ less error prone I am fine with ;)